### PR TITLE
`voltage_to_wavelength` precision and units

### PR DESCRIPTION
**Pull request by j-c-c**
This PR addresses issue #228.

The units of measure for wavelength were incorrectly indicated as nanometers in the docstring. This has been changed to angstroms. Also, added the formula being used as a comment.

Additionally, the precision of the constants used in the formula have been increased to doubles precision. This broke two hardcoded tests in `test_filters.py`. I've left the original hardcoded reference arrays and adjusted the `np.allclose` tolerance to 1e-4 on both tests (with clarifying comments).

### CODE REVIEW

File: tests/test_filters.py [hunk 1]
```diff
@@ -117,196 +117,198 @@ def testRadialCTFFilter(self):
     def testRadialCTFFilterGrid(self):
         filter = RadialCTFFilter(defocus=2.5e4)
         result = filter.evaluate_grid(8, dtype=self.dtype)
+        reference = np.array(
```
Comment (garrettwrong) [inline tests/test_filters.py]: yikes, themotivation to just adjust tol was to avoid rewriting all the lines...

**Comment by j-c-c** [comment]
There was some question about inconsistency of units in the formulation of `chi` which appears here in the `ctf_estimator`
https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/d08a9f7085efb352126784782f69d1146048f018/src/aspire/ctf/ctf_estimator.py#L127-L131

and here in the `evaluate()` method of the `CTFFilter`
https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/d08a9f7085efb352126784782f69d1146048f018/src/aspire/operators/filters.py#L437-L442

It looks like the formulations are consistent within themselves, with the assumption that the spherical aberration parameter `cs` or `Cs` is in millimeters. In that case the factor of `1e6` in the `ctf_estimator` is a conversion of the `cs` units from mm to nm, which is consistent with `lmbd` being in nm. Likewise, the factor of `1e7` in the `CTFFilter` is a conversion of mm to angstroms, which is consistent with `lmbd` being in angstroms.

**Comment by garrettwrong** [comment]
Sounds good.  Do you think it is worth converting the formulas to both be in Angstrom now that we've confirmed the units?

**Comment by j-c-c** [comment]
> Sounds good. Do you think it is worth converting the formulas to both be in Angstrom now that we've confirmed the units?

Sure. There is one hardcoded test that fails with this change. I'll give some details in the morning and we can discuss.

**Comment by j-c-c** [comment]
Ok, I've converted the units in `estimate_ctf` to angstroms and updated the reference values in the unit test. I think we're good to go here.

@garrettwrong, I do want to note that as I was updating the units I noticed one formula that appears to be off by a factor of 2:
https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/6b28cb23c22df53bf66930dbfc2aad32a44ed6a6/src/aspire/ctf/ctf_estimator.py#L854-L856

In particular, I think we need to multiply `df` by 2. I can change this here, but maybe this should be in a separate PR addressing the correctness of the formulas in `estimate_ctf`.

**Comment by garrettwrong** [comment]
> Ok, I've converted the units in `estimate_ctf` to angstroms and updated the reference values in the unit test. I think we're good to go here.
> 
> @garrettwrong, I do want to note that as I was updating the units I noticed one formula that appears to be off by a factor of 2:
> 
> https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/6b28cb23c22df53bf66930dbfc2aad32a44ed6a6/src/aspire/ctf/ctf_estimator.py#L854-L856
> 
> In particular, I think we need to multiply `df` by 2. I can change this here, but maybe this should be in a separate PR addressing the correctness of the formulas in `estimate_ctf`.

Yeah, let's just make a separate issue about the formula. Validating the CTF estimator will be a project. I think you've already done a great service to the whole codebase by getting to the bottom of that voltage_to_wavelength formula's units. Lets do the review and circle back to CTF estimator later.

File: tests/test_CtfEstimate.py [hunk 2]
```diff
@@ -18,10 +18,11 @@
 class CtfEstimatorTestCase(TestCase):
     def setUp(self):
         self.test_input_fn = "sample.mrc"
+        # Replaced test_output values after converting `estimate_ctf` units to angstroms.
         self.test_output = {
-            "defocus_u": 1.137359876e03,
-            "defocus_v": 9.617226108e02,
-            "defocus_ang": 1.5706205116381249,
+            "defocus_u": 10848.056020345417,  # replaced 1.137359876e03
```
Comment (garrettwrong) [inline tests/test_CtfEstimate.py]: take out the `# replaced`. We can use `git` for that.

**Review by garrettwrong — CHANGES_REQUESTED** [review]
Just update the comments and I think we should be good with this. Thanks!

File: tests/test_CtfEstimate.py (hunk 2 above)
Comment (j-c-c) [reply tests/test_CtfEstimate.py]: Ok, thought you might suggest that. removing.

**Comment by j-c-c** [comment]
> > Ok, I've converted the units in `estimate_ctf` to angstroms and updated the reference values in the unit test. I think we're good to go here.
> > @garrettwrong, I do want to note that as I was updating the units I noticed one formula that appears to be off by a factor of 2:
> > https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/6b28cb23c22df53bf66930dbfc2aad32a44ed6a6/src/aspire/ctf/ctf_estimator.py#L854-L856
> > 
> > In particular, I think we need to multiply `df` by 2. I can change this here, but maybe this should be in a separate PR addressing the correctness of the formulas in `estimate_ctf`.
> 
> Yeah, let's just make a separate issue about the formula. Validating the CTF estimator will be a project. I think you've already done a great service to the whole codebase by getting to the bottom of that voltage_to_wavelength formula's units. Lets do the review and circle back to CTF estimator later.

created issue #742

File: tests/test_CtfEstimate.py (hunk 2 above)
Comment (j-c-c) [reply tests/test_CtfEstimate.py]: done.

**Review by garrettwrong — APPROVED** [review]
Thanks for making that issue. LGTM.

**Pull request merged**